schemas/taskcluster-github-config.v1.yml

```diff
@@ -0,0 +1,36 @@
+$schema:            http://json-schema.org/draft-06/schema#
+title:              ".taskcluster.yml format"
+description: |
+ Description of a taskclusterrc.yml file v1, which may be used to generate a taskgraph
+ and tasks.
+type:               object
+properties:
+  version:
+    description:    ".taskcluster.yml version"
+    enum:           [1]
+    type:           integer
+  policy:
+    pullRequests:
+      description: |
+          Policy for creating tasks for pull requests.  The effective policy is found in this property
+          in the `.taskcluster.yml` file in the repository's default branch.  See the documentation for
+          detailed definition of the options.
+      type: string
+      enum:
+        - public
+        - collaborators
+  tasks:
+    type:           array
+    default:        []
+    items:
+        title:              "Task Definition"
```

Comment:
I think these will be better called something like "task definition template" maybe? They don't need to be task definitions until after this is rendered. Similarly with the `created` and `deadline` fields. Probably don't need to be asserted by this schema? I might be missing something though!

Reply:
I gave it a thunk, and it seems to me we don't have to assert anything. `created` comes easily from context, we can have some default deadline... Why did we assert them in v0?

Reply:
In hooks, we call these a "task template", so we could do the same here.

We also had to remove asserting created/deadline/expires in hooks.  The template can create all three easily enough with `$fromNow`.